plugins/system/zswap

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
```

Comment:
I think, you can use `/bin/sh` for this plugin?
(just removing the `function` keywords)

Reply:
Didn't work

Reply:
What exactly failed to work?
Maybe you did not add `()` after the function name? (this is optional in bash)
Maybe run `shellcheck -s dash`? Its output may be easier to understand, than the shell's syntax error messages.

We strive to keep the number of dependencies of munin plugins minimal (as long as it does not hurt maintainability). Something like `bash` renders plugins unusable on platforms like OpenWrt.
Thank you!

Reply:
that was it...